client/state/data-layer/third-party/directly/index.js

```diff
@@ -11,17 +15,28 @@ import {
 } from 'state/help/directly/actions';
 import * as directly from 'lib/directly';
 
-export function askQuestion( store, action, next ) {
+export function askQuestion( { dispatch }, action, next ) {
+	dispatch( recordTracksEvent( 'calypso_directly_ask_question' ) );
```

Comment:
I think we should do `directly.askQuestion( ... ).then( () => dispatch( recordTracksEvent( 'calypso_directly_ask_question' ) ) )`